### DISCUSSION TITLE
Allow for adjustable slider tick rate for Drops mid map in osu!catch editor

### DISCUSSION THREAD

**Discussion by SecreOsu** [ppy/osu#21341] [Ideas]
Currently in catch the slider tick rate can only be adjusted on a per map basis. This creates situations in which a 1/3 flow section used in a 1/2 majority map would force the mapper to currently use slider tick rate 1 as opposed to 2 or 3 because of playability/aesthetics concerns. Currently this is manipulatable by changing the BPM of the sections of the map you want to be different, but this does not abide by the ranking criteria's timing rule. Having the ability to change slider tick rate with the usage of Inherited Points would allow for a MUCH broader application of Drops in the osu!catch ranked section.

Attached first is a screenshot of a 1/3 slider using slider tick rate 2. As you can see, the Drop positioned on this slider is in a very awkward spot. Changing to slider tick rate 3 is not a viable option as the rest of the map is predominately 1/2 rhythm. Allowing for the tick rate to be adjustable to 3 to match the music would change this pattern to the second attached image, a much nicer and accurate representation of the music. This is just the most basic and common example of the use cases of this feature.
![drop](https://user-images.githubusercontent.com/80791374/202936050-4e14a49c-eeb9-4ecf-bcd0-ebfc56dc36fd.jpg)
![drop2](https://user-images.githubusercontent.com/80791374/202936056-70e67854-9fe1-49b9-aef8-ba0a5e5699a5.jpg)

**Comment by mk56-spn** [comment]
are you requesting this with the intent that it be added to lazer or stable, seeing as these screenshots are from stable, and this repo is exclusively for lazer

**Reply by SecreOsu** [reply]
> are you requesting this with the intent that it be added to lazer or stable, seeing as these screenshots are from stable, and this repo is exclusively for lazer

Lazer. I used stable screenshots for the sake of ease of access. I know stable is feature locked currently (although frankly, this ability is overdue since the existence of catch tbh)

**Comment by peppy** [comment]
To confirm, this CAN already be achieved by using bpm multiples, right?

If we are to add this in the future, it would not be a catch-specific thing. Other rulesets can also benefit from this.

**Reply by SecreOsu** [reply]
> To confirm, this CAN already be achieved by using bpm multiples, right?
> 
> If we are to add this in the future, it would not be a catch-specific thing. Other rulesets can also benefit from this.

Yes, this is already achievable by simply adjusting the bpm. The major issue is that this goes against the ranking criteria rule:

"Beatmaps must be perfectly timed. This means [BPM](https://osu.ppy.sh/wiki/en/Beatmapping/Beats_per_minute) and [offset](https://osu.ppy.sh/wiki/en/Beatmapping/Offset) are exactly synchronised with the song. Beatmaps with constantly changing BPM may be impossible to perfectly time and should instead be as accurate as possible without negatively affecting gameplay. Complex timing during breaks or spinners is optional."

Therefore not allowing any maps that use this currently to get ranked. Seeing this for all rulesets would be great, I think the only ones that benefit majorly from this are catch and standard though currently.
